<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-9U807NB" ignored="0" name="Application Test 2" tests="3" failures="1" timestamp="18 Mar 2021 03:33:15 GMT" time="0.106" errors="0">
  <testcase name="testMethod3" time="0.005" classname="com.TestNGproject.TestNGAssertsDemo"/>
  <testcase name="testMethod2" time="0.043" classname="com.TestNGproject.TestNGAssertsDemo">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [3] but found [4]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [3] but found [4]
	at com.TestNGproject.TestNGAssertsDemo.testMethod2(TestNGAssertsDemo.java:36)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testMethod2 -->
  <testcase name="testMethod1" time="0.002" classname="com.TestNGproject.TestNGAssertsDemo"/>
</testsuite> <!-- Application Test 2 -->
